code/wdb/writedown.q

```diff
@@ -2,13 +2,12 @@
 
 /-Required variables for savetables function
 compression:@[value;`compression;()];                                      /-specify the compress level, empty list if no required
-savedir:@[value;`savedir;`:temphdb];                                       /-location to save wdb data
-hdbdir:@[value;`hdbdir;`:hdb];                                             /-move wdb database to different location
+savedir:hsym @[value;`savedir;`:temphdb];                                       /-location to save wdb data
+hdbdir:hsym @[value;`hdbdir;`:hdb];                                             /-move wdb database to different location
 
-hdbsettings:(`compression`hdbdir)!(compression;hdbdir);
+hdbsettings:(`compression`hdbdir)!(compression;hsym hdbdir);
```

Comment:
are these changes necessary?

Reply:
No they are not, the only reason why I added them is that I've bumped into some errors where the code did not use hsym with these variables therefore concatenations were incorrect(.Q.dd, ` sv).
Then I looked at the purpose of these variables and as I see they only used to point to certain directories, then maybe something gets appended to them - then why not have them hsym by default? This is their only purpose.